content/js/getting-started.md

```diff
@@ -2,39 +2,92 @@
+a Skygear server ready. If you haven't got one already, you can sign up for a free
+cloud hosted server at [Skygear Portal](https://portal.skygear.io/signup).
+The deployment of your server is handled automatically so you can start using
+the JS SDK right away.
 
 <a name="about-js-sdk"></a>
 ## About the SDK
 
-The JS SDK lets you integrate your web app or
-[React Native](https://facebook.github.io/react-native/) app with Skygear
-quickly. Based on the asynchronous Promise pattern, the SDK communicates with
-the Skygear server, and allows you to perform a variety of tasks easily:
+The JS SDK allows you to quickly integrate your web or
+[React Native](https://facebook.github.io/react-native/) app with the Skygear server.
+Based on the async [promise](https://github.com/promises-aplus/promises-spec) pattern,
+you can easily perform a variety of tasks, most of the API calls will return
+a promise object:
 
-- Authenticating users
-- Saving/Querying records to/from the database with various data types
-- Setting access control for different users and roles in accessing
-  database records
-- Managing follower/following/friend relationship between users
-- Sending and receiving instant message via Publish-Subscribe (PubSub)
-- Uploading images and files
+- **Create / Authenticate Users** (see [User & Authentication]())
+```js
+skygear.signupWithUsername('alice', 'P@s$w0Rd'); // create user
+skygear.loginWithUsername('alice', 'P@s$w0Rd'); // login
+```
+
+- **Save Records** (see [Records]())
+```js
+const PersonRecord = skygear.Record.extend('person');
+const bob = new PersonRecord({ name: 'bob' });
+skygear.publicDB.save(bob);
+```
+
+- **Query Records** (see [Queries]())
+```js
+const PersonRecord = skygear.Record.extend('person');
+const findBob = new skygear.Query(PersonRecord).equalTo('name', 'bob');
+skygear.publicDB.query(findBob);
+```
+
+- **Send / Revieve Message via PubSub Channels** (see [PubSub Channels]())
+```js
+skygear.on('message', data => console.log(data)); // subscribe on channel
+skygear.pubsub.publish('message', { hello: 'world' }); // publish to channel
+```
+
+- **Manage Access Control** (see [Access Control]())
+```js
+const bob = new skygear.User({ user_id: '523e7df5-655c-4250-8df0-c38271ea53ba' });
+const MessageRecord = skygear.Record.extend('message');
+const message = new MessageRecord({ something: 'bad about Bob' });
+message.setNoAccessForUser(bob); // to talk behind Bob's back
+skygear.publicDB.save(message);
+```
+
+- **Perform Geolocational Queries** (see [Geolocation]())
+```js
+const BuildingRecord = skygear.Record.extend('building');
+const saxonTower = new BuildingRecord({ location: new skygear.Geolocation(22.3366140, 114.1486090) });
+const leroyPlaza = new BuildingRecord({ location: new skygear.Geolocation(22.3361170, 114.1474730) });
+skygear.publicDB.save([saxonTower, leroyPlaza]);
+// ----------------------------------------------------------------------
```

Comment:
Instead of using a comment separator, how about adding comments to the beginning of each section like "saving building geolocations to database" and "querying locations by geolocation distance"?